test/Game-test.js

```diff
@@ -1,19 +1,20 @@
+global.data = require('../dataset.js');
+global.domUpdates = require('../domUpdates.js');
+global.Player = require('../class/Player.js');
+global.Puzzle = require('../class/Puzzle.js');
+global.Wheel = require('../class/Wheel.js');
 const chai = require('chai');
 const expect = chai.expect;
-global.data = require('../dataset.js');
-global.domUpdates = require('../domUpdates.js')
-const Puzzle = require('../class/Puzzle.js');
-const Wheel = require('../class/Wheel.js');
 const Game = require('../class/Game.js');
-const Player = require('../class/Player.js');
 const spies = require('chai-spies');
 chai.use(spies);
 chai.spy.on(global.domUpdates, ['getPlayerNames'], () => ['first', 'second', 'third']);
 chai.spy.on(global.domUpdates, ['hideStartScreen'], () => true);
-
+chai.spy.on(global.domUpdates, ['displayPuzzle'], () => true);
```

Comment:
If `hideStartScreen` and `displayPuzzle` both return `true` for the spy, then you can combine these into one line.